test/Microsoft.VisualStudio.Web.BrowserLink.Test/TaskAssert.cs

```diff
@@ -113,22 +105,7 @@ private static void ThrowFailure(Exception exception, params string[] failureMes
         {
```

Comment:
Can this method be removed? `exception` parameter is not being used. All callers can switch to using the other overload of this method.